src/app/client/src/app/modules/shared/components/share-link/share-link.component.ts

```diff
@@ -95,6 +95,15 @@ export class ShareLinkComponent implements OnInit {
     document.execCommand('copy');
   }
 
+  setCopyContentSharingLinkInteractEdata() {
+    const searchInteractEdata = {
+      id: 'content-share-link-btn',
+      type: 'share',
```

Comment:
There is not type called share in interact event.
https://github.com/ekstep/Common-Design/wiki/Telemetry-V3#interact